app.py

```diff
@@ -262,11 +247,21 @@ def _get_repo_from_source(source):
 
 
 def _get_rust_deps_versions(quipucords_repo, quipucords_sha):
-    requirements_url = QUIPUCORDS_REQUIREMENTS_URL % (quipucords_repo, quipucords_sha)
-    requirements_content = requests.get(requirements_url).content.decode()
+    requirements_content = ""
+    for file in ["requirements.txt", "requirements-build.txt"]:
+        requirements_url = QUIPUCORDS_FILE_URL % (quipucords_repo, quipucords_sha, file)
+        requirements_content += requests.get(requirements_url).content.decode()
     versions = {}
     for dependency in RUST_CARGO_PATH.keys():
         match = re.search(rf"{dependency}==([\d\.]+)", requirements_content)
+        if not match:
+            message = f"couldn't find rust dependency '{dependency}'"
+            console.print(f":warning: [red]{message}[/red] :warning:")
+            console.print(
+                "[red]if you are not building an older version of discovery, "
+                "please check quipucords dependencies and update chaski[/red]"
+            )
+            continue
```

Comment:
How would you feel about raising an exception to force the process to exit instead of simply printing messages and returning an incomplete `versions`? In my little helper script, I assume if chaski exits with normal 0 that everything is fine to proceed, but if chaski exits non-zero, I halt and force the user (me) to investigate and take action.